jetty-server/src/main/java/org/eclipse/jetty/server/session/SessionHandler.java

```diff
@@ -136,11 +136,12 @@
 
     /**
      * Web.xml session-timeout is set in minutes, but is stored as an int in seconds by HttpSession and
-     * the sessionmanager. Thus MAX_INT is the max number of seconds that can be set, and MAX_INT/60 is the
+     * the sessionmanager. Thus MAX_INT is the max number of seconds that can be set so MAX_INT/60 is the
      * max number of minutes that you can set.
      */
-    public static final java.math.BigDecimal MAX_INACTIVE_MINUTES = new java.math.BigDecimal(Integer.MAX_VALUE / 60);
-
+    public static final int MAX_SESSION_TIMEOUT_MINS = Integer.MAX_VALUE / 60;
+    public static final int MIN_SESSION_TIMEOUT_MINS = Integer.MIN_VALUE / 60;
```

Comment:
These constants should not be in this class, as they are only ever used by `ServletContextHandler`, so they should be moved there and made `private`.